src/main/java/br/com/mercadolivre/loja/secutiry/WebSecurityConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package br.com.mercadolivre.loja.secutiry;
```

Comment:
Typo no nome do pacote

```suggestion
package br.com.mercadolivre.loja.securiyy;
```